Makefile

```diff
@@ -35,7 +35,6 @@ GIT_COMMIT = $(shell git rev-parse --short HEAD)
 AUTHOR ?= $(shell git config user.email || echo $$USER)
 
 ENABLE_METRICS ?= true
-BUILD_TAGS ?= 
```

Comment:
If you're going to remove `BUILD_TAGS` from here then you need to add them to every target that uses it.

Otherwise just leave it here, but add `export`.

Reply:
As I understand, `BUILD_TAGS` is only set outside of `Makefile` and `make` sets `$(BUILD_TAGS)` according to it.

If `BUILD_TAGS` is not set as env var, `$(BUILD_TAGS)` is empty.

Reply:
Yeah. You're right. I think it's just useful to have an explicit indication that `BUILD_TAGS` is being used by a target, so lets add an `export BUILD_TAGS ?=` to the test target to indicate that the script uses it.